packages/mdc-textfield/README.md

```diff
@@ -313,8 +305,6 @@ Method Signature | Description
 `activateFocus() => void` | Activates the focus state of the Text Field. Normally called in response to the input focus event.
 `deactivateFocus() => void` | Deactivates the focus state of the Text Field. Normally called in response to the input blur event.
 `setHelperTextContent(content: string) => void` | Sets the content of the helper text.
-`setIconAriaLabel(label: string) => void` | Sets the aria label of the icon.
```

Comment:
Removed accidentally?

Reply:
I never removed this so it must have been a bad sync on my part. I added it back!